docs/DASHBOARD_CREATION.rst

```diff
@@ -1586,7 +1586,7 @@ Optional Arguments:
 -  ``title2`` - a second line of title text
 -  ``step`` - the size of step in temperature when fading the slider up
    or down
--  ``units`` - the unit symbol to be displayed
+-  ``unit`` - the unit symbol to be displayed
```

Comment:
For consistency with `basedisplay` it should be `units`.

Reply:
Changed it to `units` in the widget so not merging this.